dvc/repo/plot.py

```diff
@@ -0,0 +1,208 @@
+import csv
+import json
+import logging
+import os
+from collections import OrderedDict
+
+from funcy import first
+from ruamel import yaml
+
+from dvc.exceptions import DvcException
+from dvc.plot import Template
+from dvc.repo import locked
+
+logger = logging.getLogger(__name__)
+
+
+class NoMetricInHistoryError(DvcException):
+    def __init__(self, path, revisions):
+        super().__init__(
+            "Could not find '{}' on any of the revisions: "
+            "'{}'".format(path, ", ".join(revisions))
+        )
+
+
+class NoMetricOnRevisionError(DvcException):
+    def __init__(self, path, revision):
+        self.path = path
+        self.revision = revision
+        super().__init__(
+            "Could not find '{}' on revision: " "'{}'".format(path, revision)
```

Comment:
Dude, what's up with colons? :smile: 
```suggestion
            "Could not find '{}' on revision '{}'".format(path, revision)
```